src/util/motionModel.ts

```diff
@@ -298,6 +307,55 @@ let emptyIterationCounter = 0;
 let prevGnssFile = '';
 let prevGpsRecord: GNSS | undefined = undefined;
 
+const createFileNameForFAT32 = (gnssFile: string, fileCreationDate: string) => {
+  //We exclude all dots and colons from the filename and replace with - for FAT32 compatibility
+  const parts = gnssFile.split('.');
+  if (parts.length > 1) {
+    const lastPart = parts.pop(); // Remove the last part
+    const replacedString = parts.join('-') + '.' + lastPart;
+    return fileCreationDate + '/' + replacedString;
+  }
+  return '';
+};
+
+const copyFileToUSB = async (fileName: string, sourceFolder: string, fileType: FILETYPE) => {
```

Comment:
I'd not pass `sourceFolder` as parameter to this function,
that wouldmake more sense if `copyFileToUSB` function knows where USB folder is located by itself.